packages/mobx-undecorate/src/undecorate.ts

```diff
@@ -399,18 +399,19 @@ export default function transform(
     }
 
     function warn(msg: string, node: Node) {
-        if (process.env.NODE_ENV === "test") {
```

Comment:
Why did you remove this check? It's not really useful to see all warnings when running tests.